src/imitation/scripts/config/train_adversarial.py

```diff
@@ -15,7 +15,7 @@
 @train_ex.config
 def train_defaults():
   env_name = "CartPole-v1"  # environment to train on
-  total_timesteps = 1e5  # Num of environment transitions to sample
+  total_timesteps = int(1e5)  # Num of environment transitions to sample
```

Comment:
Why do we need to do the conversion here? Might be simpler to cast to `int` inside `train_adversarial` so only have to do it once.

Reply:
fair, I've added an int cast as the first statement of train_adversarial. (Could also change the type of `total_timesteps` to `Union[float, int]` or something, but that felt excessive.)